tests/integration/llm/client.py

```diff
@@ -192,6 +192,34 @@ def get_model_name():
     },
 }
 
+transformers_neuronx_neo_model_spec = {
+    "llama-2-13b-rb": {
+        "seq_length": [1024],
+        "batch_size": [1, 4],
+        "tokenizer": "TheBloke/Llama-2-13B-fp16"
+    },
+    "mixtral-8x22b": {
+        "workers": 1,
+        "seq_length": [512],
+        "batch_size": [2]
```

Comment:
do we have tokenizer defined here as well?

Reply:
@a-ys

Reply:
My understanding is: only included tokenizer for rolling batch handler tests since those use awscurl to test. `‎test_transformers_neuronx_handler` doesn't use tokenizer.